packages/client/webapp/package.json

```diff
@@ -23,8 +26,10 @@
 		"@playwright/experimental-ct-vue": "1.25.2",
 		"@playwright/test": "1.25.2",
 		"@rushstack/eslint-patch": "1.2.0",
+		"@types/lodash": "4.14.186",
 		"@vitejs/plugin-vue": "3.1.0",
 		"@vitest/coverage-c8": "0.23.4",
+		"sass": "1.55.0",
```

Comment:
I believe @YohannParis wants to avoid this but will let him weigh in on that

Reply:
We are adding it now, to get the data-explorer in, but it is to be removed!